roles/dpul/meta/main.yml

```diff
@@ -16,3 +16,16 @@ galaxy_info:
 
 dependencies:
   - role: shared_data
+  - role: ruby
+  - role: deploy_user
+  - role: passenger
+  - role: redis
+  - role: postgresql
+  - role: nodejs
+  - role: openjdk
+  - role: imagemagick
+  - role: rails_app
+  # TODO: THis role runs with the line below commented out.  We need the line below to work
+  # - {role: solr, when: postgresql_is_local == true}
```

Comment:
Do we need this line to work? Is that for tests?

Reply:
I think this line is an error. Is there any reason one would install postgresql locally for dpul?

Reply:
dpul connects to the solrcloud. I think this line can simply be deleted.